nixos/modules/virtualisation/oci-containers.nix

```diff
@@ -313,6 +327,9 @@ let
       # ExecReload = ...;
       ###
 
+      Environment=if cfg.backend == "podman" then "PODMAN_SYSTEMD_UNIT=podman-${name}.service" else {};
+      Type=if cfg.backend == "podman" then "notify" else {};
+      NotifyAccess=if cfg.backend == "podman" then "all" else {};
```

Comment:
This could have used optionalAttrs and ideally there should be a space before and after the first equal sign.

Reply:
@SuperSandro2000 mind sending a PR?

Reply:
Half done in 76e1e908c1cbcf54b5a328af6e62a483f0637a60 already